data/json/recipes/recipe_weapon.json

```diff
@@ -1400,6 +1400,69 @@
   ]
 },{
   "type" : "recipe",
+  "result": "pipe_combination_gun",
+  "id_suffix": "using mod",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_RANGED",
+  "skill_used": "mechanics",
+  "skills_required": [ "gun", 1 ],
+  "difficulty": 4,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities":[
+    {"id":"SAW_M_FINE","level":1,"amount":1},
+    {"id":"SCREW_FINE","level":1,"amount":1}
+  ], "components": [
+    [
+      [ "pipe", 1 ]
+    ],
+    [
+      [ "nail", 1 ]
+    ],
+    [
+      [ "spring", 1 ]
+    ],
+    [
+      [ "scrap", 2 ]
+    ],
+    [
+      [ "2x4", 1 ]
+    ],
+    [
+      [ "combination_gun_shotgun_pipe", 1 ]
```

Comment:
This recipe results in misleading description: it appends "This can be used to craft pipe gun" to description of combination pipe gun (NOT just the mod itself).